packages/paste-website/src/pages/components/heading/index.mdx

```diff
@@ -82,7 +82,8 @@ Headings should allow users to easily distinguish content sections on the page.
 - Headings should be used with intention that maintains a coherent document outline
 - Don’t use heading tags to resize text. Instead use the `<Text>` primitive
 - Don’t skip headline levels. Always start with `<h1>`, `<h2>`, and so on
-- There can only be one `<h1>` tag per page.
+- There can only be one `<h1>` tag per page
+- Always use sentence case when writing headings, for more information read the [product style guide](/foundations/content/product-style-guide/#sentence-case)
```

Comment:
I don't see this bullet point in the preview, though I do see the other changes. Is this just me?

Reply:
Ooooh - the preview probably hasn't been updated yet